packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -1829,6 +1829,10 @@
             "properties": {
               "id": {
                 "type": "keyword"
+              },
+              "hash": {
+                "type": "keyword",
+                "index": false
```

Comment:
`hash` field should be excluded from SO mappings as it is not powering any search.

Reply:
I might be missing something here, maybe @juliaElastic can help me clarify.

As I understand, this means removing the `hash` field from `x-pack/plugins/fleet/server/saved_objects/index.ts` altogether. This makes sense as we're not searching on the hash, but I'm unsure whether we'll be able to retrieve the hash in all scenarios. FWIW I've tested the repro steps locally and it does work.

Reply:
Solved via Slack - for reference, this tutorial was useful: https://github.com/elastic/kibana/blob/main/dev_docs/tutorials/saved_objects.mdx.